# 参考文献

1. [Cell RangerTM R Kit Tutorial: Secondary Analysis on 10x GenomicsTM Single Cell 3' RNA-seq PBMC Data.](https://cf.10xgenomics.com/supp/cell-exp/cellrangerrkit-PBMC-vignette-knitr-1.1.0.pdf)
2. [Dobin A, Davis C A, Schlesinger F, et al. STAR: ultrafast universal RNA-seq aligner\[J\]. Bioinformatics, 2013, 29(1): 15-21.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905/)
3. [Butler, Andrew et al. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Nature biotechnology vol. 36,5 (2018): 411-420. doi:10.1038/nbt.4096](https://pubmed.ncbi.nlm.nih.gov/29608179/)
4. [Stuart, Tim et al. Comprehensive Integration of Single-Cell Data. Cell vol. 177,7 (2019): 1888-1902.e21. doi:10.1016/j.cell.2019.05.031](https://pubmed.ncbi.nlm.nih.gov/31178118/)
5. [Van der Maaten, L.J.P. Accelerating t-SNE using Tree-Based Algorithms. Journal of Machine Learning Research 15, 3221-3245 (2014).](https://jmlr.org/papers/v15/vandermaaten14a.html)
6. [McInnes, L, Healy, J, UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction, ArXiv e-prints 1802.03426, 2018](https://arxiv.org/abs/1802.03426)
7. [Aran, Looney, Liu et al. Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage. Nature Immunology (2019)](https://www.nature.com/articles/s41590-018-0276-y)
8. [Deng YY, Li JQ, Wu S F, Zhu YP, et al. Integrated NR Database in Protein Annotation System and Its Localization. \[Computer Engineering Italic\] 2006.,32(5):71-74.](http://en.cnki.com.cn/Article\_en/CJFDTOTAL-JSJC200605025.htm)
9. [Apweiler R, Bairoch A, Wu CH, Barker WC, et al. UniProt: the Universal Protein knowledgebase. \[Nucleic Acids Research Italic\] 2004 Jan 1;32(Database issue):D115-9.](http://tour.biocloud.net/article/v1/into/articleDetail/27899622)
10. [Ashburner M, Ball C A, Blake J A, Botstein D, et al. Gene ontology: tool for the unification of biology. \[Nature Genetics Italic\] 2000, 25(1): 25-29.](http://tour.biocloud.net/article/v1/into/articleDetail/10802651)
11. [Tatusov RL, Galperin MY, Natale DA. The COG database: a tool for genome scale analysis of protein functions and evolution. \[Nucleic Acids Research Italic\] 2000, 28(1):33-36.](http://tour.biocloud.net/article/v1/into/articleDetail/10592175)
12. [Koonin EV, Fedorova ND, Jackson JD, et al. A comprehensive evolutionary classification of proteins encoded in complete eukaryotic genomes. Genome biology. 2004, 5(2): R7.](http://tour.biocloud.net/article/v1/into/articleDetail/14759257)
13. [Finn RD, Bateman A, Clements J, et al. Pfam: the protein families database. Nucleic acids research. 2013: gkt1223.](http://tour.biocloud.net/article/v1/into/articleDetail/18039703)
14. [Kanehisa M, Goto S, Kawashima S, Okuno Y, et al. The KEGG resource for deciphering the genome. \[Nucleic Acids Research Italic\] 2004, 32(Database issue):D277-D280.](http://tour.biocloud.net/article/v1/into/articleDetail/14681412)
15. [Joshi-Tope G, Gillespie M, Vastrik I, D’Eustachio P, Schmidt E, de Bono B, Jassal B, Gopinath GR, Wu GR, Matthews L, Lewis S, Birney E, Stein L. Reactome: a knowledgebase of biological pathways.Nucleic Acids Res.1:D428-32.](https://academic.oup.com/nar/article/33/suppl\_1/D428/2505340)
16. [monocle-release](http://cole-trapnell-lab.github.io/monocle-release/docs/)
17. [Wu, Xunyao et al. Single-cell sequencing of immune cells from anticitrullinated peptide antibody positive and negative rheumatoid arthritis. Nature communications vol. 12,1 4977. 17 Aug. 2021, doi:10.1038/s41467-021-25246-7](https://pubmed.ncbi.nlm.nih.gov/34404786/)
18. [La Manno, Gioele et al. RNA velocity of single cells. Nature vol.560,7719 (2018): 494-498. doi:10.1038/s41586-018-0414-6](https://pubmed.ncbi.nlm.nih.gov/30089906/)
19. [Bergen, Volker et al. Generalizing RNA velocity to transient cell states through dynamical modeling. Nature biotechnology vol. 38,12 (2020):1408-1414. doi:10.1038/s41587-020-0591-3](https://pubmed.ncbi.nlm.nih.gov/32747759/)
20. [Efremova, Mirjana et al. CellPhoneDB: inferring cell-cell communication from combined expression of multi-subunit ligand-receptor complexes. Nature protocols vol. 15,4 (2020): 1484-1506. doi:10.1038/s41596-020-0292-x](https://pubmed.ncbi.nlm.nih.gov/32103204/)
21. [Subramanian, Aravind et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.Proceedings of the National Academy of Sciences of the United States of America vol. 102,43 (2005): 15545-50. doi:10.1073/pnas.0506580102](https://pubmed.ncbi.nlm.nih.gov/16199517/)
22. [Chen, Zhaohui et al. Single-cell RNA sequencing highlights the role of inflammatory cancer-associated fibroblasts in bladder urothelial carcinoma. Nature communications vol. 11,1 5077. 8 Oct. 2020,doi:10.1038/s41467-020-18916-5](https://pubmed.ncbi.nlm.nih.gov/33033240/)
23. [Hänzelmann, Sonja et al. GSVA: gene set variation analysis for microarray and RNA-seq data. BMC bioinformatics vol. 14 7. 16 Jan. 2013,doi:10.1186/1471-2105-14-7](https://pubmed.ncbi.nlm.nih.gov/23323831/)
24. [Chen, Yu-Pei et al. Single-cell transcriptomics reveals regulators underlying immune cell diversity and immune subtypes associated with prognosis in nasopharyngeal carcinoma. Cell research vol. 30,11 (2020): 1024-1042.doi:10.1038/s41422-020-0374-x](https://pubmed.ncbi.nlm.nih.gov/32686767/)
25. [inferCNV](https://github.com/broadinstitute/inferCNV)
26. [Puram, Sidharth V et al. Single-Cell Transcriptomic Analysis of Primary and Metastatic Tumor Ecosystems in Head and Neck Cancer. Cell vol. 171,7(2017): 1611-1624.e24. doi:10.1016/j.cell.2017.10.044](https://pubmed.ncbi.nlm.nih.gov/29198524/)
27. [He, Shuai et al. Single-cell transcriptome profiling of an adult human cell atlas of 15 major organs. Genome biology vol. 21,1 294. 7 Dec. 2020,doi:10.1186/s13059-020-02210-0](https://pyscenic.readthedocs.io/en/latest/index.html)
28. [Langfelder, Peter, and Steve Horvath. WGCNA: an R package for weighted correlation network analysis. BMC bioinformatics vol. 9 559. 29 Dec. 2008,doi:10.1186/1471-2105-9-559](https://pubmed.ncbi.nlm.nih.gov/19114008/)
29. [P. Langfelder, S. Horvath, WGCNA: An R package for weighted correlation network analysis. BMC Bioinformatics 9, 559 (2008) - PubMed - PMC - DOI](https://pubmed.ncbi.nlm.nih.gov/19114008/)
